From 4cf033993674da862d8e064a729ce3fb7746e195 Mon Sep 17 00:00:00 2001
From: josalaito <josalaito@hotmail.es>
Date: Mon, 23 Sep 2013 16:10:10 +0200
Subject: [PATCH] fix-hwcomposer-compile

---
 libhwcomposer/hwc_utils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libhwcomposer/hwc_utils.h b/libhwcomposer/hwc_utils.h
index c9ab627..e2a896b 100644
--- a/libhwcomposer/hwc_utils.h
+++ b/libhwcomposer/hwc_utils.h
@@ -33,6 +33,7 @@
 #define FINAL_TRANSFORM_MASK 0x000F
 #define MAX_NUM_DISPLAYS 4 //Yes, this is ambitious
 #define MAX_NUM_LAYERS 32
+#define HWC_BLIT 4
 
 //Fwrd decls
 struct hwc_context_t;
-- 
1.8.3.msysgit.0

